backup_and_restore/application_backup_and_restore/aws-sts/oadp-aws-sts.adoc

```diff
@@ -0,0 +1,43 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="oadp-aws-sts"]
+= Backing up applications on AWS STS using OADP
+include::_attributes/common-attributes.adoc[]
+:context: oadp-aws-sts-backing-up-applications
+
+toc::[]
+
+You install the OpenShift API for Data Protection (OADP) with Amazon Web Services (AWS) by installing the OADP Operator. The Operator installs link:https://{velero-domain}/docs/v{velero-version}/[Velero {velero-version}].
+
+include::snippets/oadp-mtc-operator.adoc[]
+
+You configure AWS for Velero, create a default `Secret`, and then install the Data Protection Application. For more details, see xref:../../..//backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator.adoc#oadp-installing-operator-doc[Installing the OADP Operator].
+
+To install the OADP Operator in a restricted network environment, you must first disable the default OperatorHub sources and mirror the Operator catalog. See xref:../../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks] for details.
+
+You can install OADP on an AWS Security Token Service (AWS STS) cluster manually. Amazon AWS provides AWS Security Token Service (AWS STS) as a web service that enables you to request temporary, limited-privilege credentials for users. You use STS to provide trusted users with temporary access to resources via API calls, your AWS console or the AWS command line interface (CLI).
+
+Before installing {oadp-first}, you must set up role and policy credentials for OADP so that it can use the {aws-full} API.
+
+This process is performed in the following two stages:
+
+. Prepare {aws-short} credentials
+. Install the OADP Operator and give it an IAM role
+
+include::modules/preparing-aws-sts-credentials-for-oadp.adoc[leveloffset=+1]
+
+include::modules/oadp-setting-resource-limits-and-requests.adoc[leveloffset=+2]
+
+include::modules/installing-oadp-aws-sts.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html/operators/user-tasks#olm-installing-from-operatorhub-using-web-console_olm-installing-operators-in-namespace[Installing from OperatorHub using the web console].
+* link:https://docs.openshift.com/container-platform/4.15/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.html[Backing up applications]
```

Comment:
xref - please

Reply:
I think we tried using an xref when we published ROSA but it didn't work which is why we had to use a hard-link instead.

Reply:
Look at the path.

The xref should be easy and will work

Reply:
@anarnold97 
Thank you for your help. Your suggestion works.